tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
```

Comment:
We can omit as we're not actually having TS output any files.

Reply:
We probably actually _want_ TS to output definition files, right? ([TS docs](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html))